### PR TITLE
Adding Tunnel1/2 Inside CIDR and Preshared Key (#6)

### PR DESCRIPTION
**Pull request by bequiet88**
Added ability to the module to handle static tunnel ips and preshared keys as discussed in the issue thread. Open for feedback.

### CODE REVIEW

**Comment by bequiet88** [comment]
will resolve merge conflict asap.

File: variables.tf [hunk 1]
```diff
@@ -44,3 +44,23 @@ variable "vpn_connection_static_routes_destinations" {
...
+  default     = ""
+}
+
+variable "tunnel1_preshared_key" {
+  description = "The preshared key of the first VPN tunnel."
+  default     = ""
+}
+
+variable "tunnel2_preshared_key" {
+  description = "The CIDR block of the inside IP addresses for the second VPN tunnel. "
```
Comment (antonbabenko) [inline variables.tf]: Wrong description on this one

File: main.tf [hunk 2]
```diff
@@ -1,34 +1,102 @@
...
 resource "aws_vpn_connection" "default" {
-  count = "${var.create_vpn_connection ? 1 : 0}"
+  count               = "${var.create_vpn_connection && length(var.tunnel1_inside_cidr) == 0  && length(var.tunnel2_inside_cidr) == 0 && length(var.tunnel1_preshared_key) == 0 && length(var.tunnel2_preshared_key) == 0 ? 1 : 0}"
 
   vpn_gateway_id      = "${var.vpn_gateway_id}"
   customer_gateway_id = "${var.customer_gateway_id}"
   type                = "ipsec.1"
 
-  static_routes_only = "${var.vpn_connection_static_routes_only}"
+  static_routes_only  = "${var.vpn_connection_static_routes_only}"
```
Comment (antonbabenko) [inline main.tf]: Not related to your PR in general, but _while you are here_:
Add [this .pre-commit-config.yml](https://github.com/terraform-aws-modules/terraform-aws-security-group/blob/master/.pre-commit-config.yaml) to the root directory and run `pre-commit install && pre-commit run -a` to format all code properly automatically.

**Comment by bequiet88** [comment]
Hi @antonbabenko,
I've tried to please all your request. Pls have a look again.
Thanks!

**Comment by bequiet88** [comment]
Updated Readme and complete example.

**Comment by antonbabenko** [comment]
[v1.2.0](https://github.com/terraform-aws-modules/terraform-aws-vpn-gateway/releases/v1.2.0) has been released.

**Pull request merged**